### PR TITLE
WIP: 2021 04 12 deeamon128 something new

### PR DESCRIPTION
**Pull request by deeamon128**

### CODE REVIEW

**Comment by swoopstyle** [comment]
💯

**Review by swoopstyle — APPROVED** [review]
💯

**Review by biggicode — APPROVED** [review]

File: web/deeamon128/html/something-new/index.html [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    <ul>
+      <li><a href="#">Home</a></li>
+      <li><a href="#">News</a></li>
+      <li><a href="#">Observing</a></li>
+      <li><a href="#">Community</a></li>
+      <li><a href="#">Subscribe</a></li>
+    </ul>
+  </div>
+
+  <script src="app.js"></script>
```
Comment (QuintyHH) [inline web/deeamon128/html/something-new/index.html]: Next time, please add the script tag in the \<head> section, and defer it: 
```
<head>
   ...........
   <script src="app.js" defer></script>
 </head>
```
 
You can read more about deferring at https://www.w3schools.com/tags/att_script_defer.asp

**Review by QuintyHH — APPROVED** [review]
Please read up my comment on script tags :)

**Review by vasilemaftita — APPROVED** [review]
Nice.

File: web/deeamon128/html/something-new/index.html (hunk 1 above)
Comment (deeamon128) [reply web/deeamon128/html/something-new/index.html]: Thank you. I will try now to see the difference. I always thought I that have to put it in the <body> section.

**Pull request merged**